src/java/test/org/apache/zookeeper/server/persistence/FileTxnLogTest.java

```diff
@@ -17,9 +17,13 @@
  */
 package org.apache.zookeeper.server.persistence;
 
-import org.apache.zookeeper.ZKTestCase;
-import org.apache.zookeeper.ZooDefs;
+import org.apache.zookeeper.*;
+import org.apache.zookeeper.data.Stat;
```

Comment:
import *  ?

Reply:
Hi @maoling do you have any more comments or I should proceed with PR. Basically I am waiting on your review :)